src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIUsersService.java

```diff
@@ -152,6 +153,7 @@ public AjaxCreateItemSuccessResponse createUser(UserCreateRequest userCreateRequ
 
 			PasswordReset passwordReset = null;
 			try {
+				user.setUserType(UserType.TYPE_LOCAL);
```

Comment:
I think this can be removed as the default initializer for `User` already sets this to `TYPE_LOCAL`